build-configs/BuildConfigType.ts

```diff
@@ -38,6 +38,7 @@ export type FeatureFlagsType = FixedCityType & {
   // Enables the option for users to suggest Integreat to their city and uses this template for the suggestion
   cityNotCooperating?: boolean
   cityNotCooperatingTemplate: string | null
+  chatbot: boolean
```

Comment:
🔧  Can we call it `chatbotEnabled` just to make it clearer elsewhere that it's a boolean and not which chatbot it is?

Reply:
I disagree, this matches our previous flags and I think its pretty clear due to it being in the namespace `featureFlags`.

Reply:
i renamed it to `chat` since i renamed chatbot to chat everywhere